### ISSUE TITLE
metrics-graphics is not working with me.. What is wrong in my code? 

### ISSUE THREAD

**Issue by AsMoh** [mozilla/metrics-graphics#481]
Hi,

I trird to use this awasome metrics-graphics, but it did not work. I am new to jquery .. 

This is the code inside my script.js --

$(document).ready(function(){

d3.json('lib/fake_users1.json', function(data) {
    data = MG.convert.date(data, 'date');
    MG.data_graphic({
        title: "Line Chart",
        description: "This is a simple line chart. You can remove the area portion by adding area: false to the arguments list.",
        data: data,
        width: 600,
        height: 200,
        right: 40,
        target: document.getElementById('#fake_users1'),
        x_accessor: 'date',
        y_accessor: 'value'
    });
});
});

and inside my html -

<div id="fake_users1">      
  </div>

However, I am getting these errors as shown in the following image. I did include the libraries(jquery , d3 and metrics-graphics). And the path for my json file is right. 

 
![error](https://cloud.githubusercontent.com/assets/10277729/9158846/c83f11c4-3f2e-11e5-8e15-b579b66c41fc.jpg)

Could you help me to solve this problem, please? Thank you in advance.

**Comment by hamilton** [comment]
Hello - 

I think this SO question might help: http://stackoverflow.com/questions/8449716/cross-origin-requests-are-only-supported-for-http-but-its-not-cross-domain

tldr: doesn't have to do with MG, but with cross origin request issues using file:/

**Comment by AsMoh** [comment]
Okk ... Thank you
